cpp/src/arrow/array-test.cc

```diff
@@ -33,15 +33,12 @@ namespace arrow {
 
 class TestArray : public ::testing::Test {
  public:
-  void SetUp() {
-    pool_ = default_memory_pool();
-  }
+  void SetUp() { pool_ = default_memory_pool(); }
```

Comment:
Interesting, is this configurable? (I prefer the old style, but it's not a dealbreaker)